book/src/terminology.md

```diff
@@ -50,6 +50,10 @@ with a ledger interpretation that matches the leader's.
 
 A [ledger](#ledger) derived from common entries but then diverged.
 
+#### fortnight
```

Comment:
don't use fortnight, `turn` is better.  but maybe to generic.  fornight is just my placeholder for a yet to be defined term